lib/svgo.js

```diff
@@ -20,7 +20,7 @@ var SVGO = module.exports = function(config) {
     this.config = CONFIG(config);
 };
 
-SVGO.prototype.optimize = function(svgstr) {
+SVGO.prototype.optimize = function(svgstr, info) {
```

Comment:
Yes, the file path must be somehow injected into the config object for each file (or set to empty/false/null for a string input): https://github.com/strarsis/svgo/blob/67e5a3ef10e349bded15bc9f508596252c2884be/lib/svgo/coa.js#L437